Objects/EventItem/Mag.gd

```diff
@@ -0,0 +1,45 @@
+extends "res://Objects/EventItem/EventItem.gd"
+
+onready var BONUS_TIME = 10
+
+var timer
+var default_damage_multiplier
+var default_damage
+
+
+func get_reward():
+	var stats = get_parent().find_node("Elf").stats
+	default_damage_multiplier = stats.damage_multiplier
+	stats.damage_multiplier *= 4.0
+	
+	var damage_stat = stats.get_stat("bows_knowledge")
+	damage_stat.set_default_value(damage_stat.default_value * stats.damage_multiplier)
```

Comment:
It's very risky way to do this. If player (for example) close the game before `delayed_reset` will be called, the changed default value will be saved. I recomend to do this via stat changers, as we did here: https://github.com/Ajver/EvD/blob/master/Scripts/ElfStats.gd#L149

But still, this changer should be removed from damage stat before exiting the program, or after program runs.

Reply:
Yes, this is very problematic. Player always can click alt+f4 but I think we should secure game when player click "exit game".
Now i create card on trello about this problem but I can fix this problem now and alone.